apps/demos/Demos/Charts/StepLine/description.md

```diff
@@ -1,2 +1,2 @@
-_Step line_ series are much like traditional line series. The primary difference is&nbsp;that in _step line_ series, data points are connected by&nbsp;vertical and horizontal lines --- resulting in&nbsp;a&nbsp;&laquo;stair step&raquo; effect.
+The DevExtreme Chart component supports step line charts. To display a step line chart, set **series**.[type](/Documentation/ApiReference/UI_Components/dxChart/Configuration/series/#type) to *"stepline"*. Step line charts connect points with vertical and horizontal lines that form a "stair step" effect. You can utilize the [StepLine](/Documentation/ApiReference/UI_Components/dxChart/Series_Types/StepLineSeries/) series type to display data with discrete values.
```

Comment:
We can shorten this paragraph. I am using this option as a basis: http://tw.devexpress.devx/documentation/guidelines/technical-writing-guidelines/demo-and-example-descriptions/#example-demo-description_1

```suggestion
Step line charts display data as points joined by horizontal and vertical line segments that form a step-like pattern. To create a step line chart, set the **series**.[type](/Documentation/ApiReference/UI_Components/dxChart/Configuration/series/#type) property to *"stepline"*. 
```

As for discrete data, could you please explain what you were trying to say in a greater detail? For now it looks like a repetition to me.